src/btree/bt_sync_obsolete.c

```diff
@@ -8,21 +8,25 @@
 
 #include "wt_internal.h"
 
-#define WT_CHECKPOINT_CLEANUP_FILE_INTERVAL 1 /* 1 second */
+#define WT_CC_FILE_INTERVAL 1 /* 1 second */
+#define WT_CC_MAX_PAGES_PER_BTREE 500
 #define WT_URI_FILE_PREFIX "file:"
 
 /*
- * __sync_obsolete_inmem_evict --
+ * __sync_obsolete_inmem_evict_or_mark_dirty --
```

Comment:
I think we should rephrase the header of that function:
```
This function checks whether the in-memory ref contains obsolete information by:
- Checking if the ref is obsolete using the newest stop time. In this case, mark it for urgent eviction.
- Checking if the ref contains obsolete time window information using the newest durable time. In this case, mark the ref dirty so reconciliation removes the obsolete data.
```

Reply:
Thanks, fixed it by moving those statements inside the function where we check for obsolete.